frontend/src/components/DropboxChooser/index.tsx

```diff
@@ -0,0 +1,68 @@
+import loadable from "@loadable/component";
+import React, { FC, useState } from "react";
+
+declare global {
+  interface Window {
+    Dropbox: {
+      isBrowserSupported: () => boolean;
+      choose: (options: unknown) => void;
+    };
+  }
+}
+
+const AsyncContent = loadable(
+  () =>
+    /*webpackChunkName: 'DropboxChooser/Content' */ import(
+      "src/components/DropboxChooser/components/Content"
+    )
+);
+
+type DropboxFile = {
+  link: string;
+};
+
+const DROPBOX_OPTIONS = {
+  extensions: [".h5ad"],
+  sizeLimit: 30 * 1024 * 1024 * 1024, // 30GB
```

Comment:
2^30 == 1 GB if that's less confusing

Reply:
Oh super great feedback 😆 Thanks a lot! Done!